rotkehlchen/constants/timing.py

```diff
@@ -19,3 +19,5 @@
 DEFAULT_READ_TIMEOUT = 30
 DEFAULT_TIMEOUT_TUPLE = (DEFAULT_CONNECT_TIMEOUT, DEFAULT_READ_TIMEOUT)
 ENS_UPDATE_INTERVAL = DAY_IN_SECONDS
+
+ETH_PROTOCOLS_CACHE_REFRESH = DAY_IN_SECONDS * 3
```

Comment:
Why is this different to `CURVE_POOLS_UPDATE_SECS` which is 7 days?

Reply:
I've modified it. Changed it to 3 days since 7 is too much to not update the registry